### DISCUSSION TITLE
Where is the HomeKit QR code for my Amcrest camera(s)?

### DISCUSSION THREAD

**Discussion by Scenario** [koush/scrypted#717] [Q&A] [answered]
### Discussed in https://github.com/koush/scrypted/discussions/611

<div type='discussions-op-text'>

<sup>Originally posted by **Scenario** March  8, 2023</sup>
In https://github.com/koush/scrypted/wiki/HomeKit-Secure-Video-Setup, Step 6 says:

> In the Camera's Scrypted settings, scan the QR code in the HomeKit app on iOS or Mac.
Do not use the QR code in the the HomeKit Plugin, this plugin is a bridge for bridging non-camera devices like switches, thermostats, etc. Cameras are run in standalone accessory mode by default, and are not bridged.

I can only find the QR code for the HomeKit Plugin. I have my first Amcrest camera configured and I see it's image (static?) but I don't see a different QR code anywhere in the Scrypted UI.

What do you mean, "In the Camera's Scripted settings?" We need an explicit path here.
</div>

**Comment by mikeburgh** [comment, marked as answer]
If you go to the device, then on the right in settings, go to extensions, tick homekit in the extensions list and press save. 

Then the homekit button should appear in the settings section, click that, and go to pairing, the QR code should be there.
